opensrp-chw-core/src/main/java/org/smartregister/chw/core/custom_views/NavigationMenu.java

```diff
@@ -99,6 +116,11 @@ public static NavigationMenu getInstance(Activity activity, View parentView, Too
                 }
 
                 SyncStatusBroadcastReceiver.getInstance().addSyncStatusListener(instance);
```

Comment:
Did you consider this implementation from client-core? [SyncStatusBroadcastReceiver](https://github.com/opensrp/opensrp-client-core/blob/master/opensrp-app/src/main/java/org/smartregister/receiver/SyncStatusBroadcastReceiver.java)

Reply:
The `SyncProgressBroadcastReceiver` is a sync progress broadcast client-core implementation and not a sync status per se. Maybe this can be improved in client-core